src/grammar.js

```diff
@@ -43,7 +43,29 @@ class Grammar {
    * @return New Grammar object that represents
    */
   static fromNFA (nfa) {
-
+    const firstSymbol = nfa.start
+    const productions = {}
+    let aux
```

Comment:
Aux variable can be a const and inside the loop;
Aux can also be a Set, removes the necessity of `indexOf() === -1`

Reply:
```
Object representation:
    {
      'S': ['aA', 'a'],
      'A': ['aS']
    }
```
if i put aux inside the loop i'll be redeclaring it every iter.